node/tests/ServerModules.test.ts

```diff
@@ -761,6 +761,97 @@ describe("Server Module Tests", () => {
                 ).toBeNull();
             });
 
+            it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+                "json.clear tests",
+                async () => {
+                    client = await GlideClusterClient.createClient(
+                        getClientConfigurationOption(
+                            cluster.getAddresses(),
+                            protocol,
+                        ),
+                    );
+                    const key = uuidv4();
+                    const jsonValue = {obj:{a:1, b:2}, arr:[1,2,3], str: "foo", bool: true, int: 42, float: 3.14, nullVal: null};
+
+                    expect(
```

Comment:
your tests could include more comments to discuss what you're testing.  it's not always obvious to a developer/reader

Reply:
Good point. I copied these tests from Python, so I don't really know what the original intent was either.